api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -84,9 +84,9 @@ public void setVersion(String version) {
         this.version = version;
     }
 
-    @Description("The KRaft metadata.version that should be used by this Kafka cluster. " +
-            "This field will be ignored when running in ZooKeeper mode. " +
-            "Defaults to {DefaultKafkaMetadataVersion}.")
+    @Description("The KRaft `metadata.version` used by the Kafka cluster. " +
```

Comment:
I would probably call it just metadata version as the naming is confusing in Kafka itself.

```suggestion
    @Description("The KRaft metadata version used by the Kafka cluster. " +
```